### PR TITLE
Update versions of python tools - fix tauolapp source location

### PR DESCRIPTION
**Pull request by davidlange6**

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
i probably have one more update later - i'll restart tests when I'm more likely stopped

**Comment by davidlange6** [comment]
please test

File: tauolapp.spec [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ### RPM external tauolapp 1.1.5
-Source: http://service-spi.web.cern.ch/service-spi/external/MCGenerators/distribution/tauola++/tauola++-%{realversion}-src.tgz
+Source: http://tauolapp.web.cern.ch/tauolapp/resources/TAUOLA.1.1.5/TAUOLA.%{realversion}-LHC.tar.gz
```
Comment (smuzaffar) [inline tauolapp.spec]: @davidlange6 ,please use `TAUOLA.%{realversion}` instead of `TAUOLA.1.1.5`

Comment (davidlange6) [reply tauolapp.spec]: whoops... thanks for noticing that

**Comment by davidlange6** [comment]
Of the unit test errors - I presume that this one is not related to the PR

stat: cannot stat 'dqm_empty.root': No such file or directory
read_missing_file_cfg.py ------------------------------------------------------------
07-Feb-2019 23:58:49 CET  Initiating request to open file file:dqm_missing.root
----- Begin Fatal Exception 07-Feb-2019 23:58:49 CET-----------------------
An exception of category 'FileOpenError' occurred while
   [0] Calling InputSource::readFile_
   [1] Opening DQM Root file
   [2] Calling StorageFactory::open()
   [3] Calling File::sysopen()
Exception Message:
Failed to open the file 'dqm_missing.root'
Input file file:dqm_missing.root was not found, could not be opened, or is corrupted.
   Additional Info:
      [a] open() failed with system error 'No such file or directory' (error code 2)
----- End Fatal Exception -------------------------------------------------


I'm fixing the other two

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Comment by smuzaffar** [comment]
@fabiocos , we overlooked this PR yesterday. If it still time in pre2 then can we get this in?

**Comment by davidlange6** [comment]
I would suggest merging this after pre2 unless pre2 is known to be delayed for several more days.. i should not affect anything, but it can.

**Comment by fabiocos** [comment]
@davidlange6 @smuzaffar I wanted to clarify today at the ORP. While cms-sw/cmssw#25890 seems orthogonal to the standard workflows, this update may be less innocent. As I hope to converge asap to pre2, I would keep a decision about its merge for this evening.

**Comment by davidlange6** [comment]
it won't affect standard workflows. but perhaps some tests (If i missed something). its just a matter of not wanting to delay pre2 for non-critical path things. (but given 48 hours more of delay, then this PR is a good target)

**Comment by fabiocos** [comment]
+1

**Pull request merged**